auto_dev/handler/exceptions.py

```diff
@@ -0,0 +1,5 @@
+"""Exceptions for the handler module."""
+
+
+class ScaffolderError(Exception):
```

Comment:
Don't we already have an exceptions path?
We don't need a separate file just for the handler exceptiond

Reply:
https://github.com/8ball030/auto_dev/pull/369